tests/sentry/tasks/test_backfill_seer_grouping_records.py

```diff
@@ -1036,6 +1059,8 @@ def test_backfill_seer_grouping_records_multiple_batches(
                     "cohort": None,
                     "last_processed_project_index": 0,
                     "only_delete": False,
+                    "skip_processed_projects": True,
+                    "skip_project_ids": None,
```

Comment:
This `test_backfill_seer_grouping_records_multiple_batches` shows we're sending the option between batches now